x-pack/plugins/fleet/common/services/license.ts

```diff
@@ -53,6 +53,13 @@ export class LicenseService {
       this.licenseInformation?.hasAtLeast('enterprise')
     );
   }
+  public isPlatinium() {
+    return (
+      this.licenseInformation?.isAvailable &&
+      this.licenseInformation?.isActive &&
+      this.licenseInformation?.hasAtLeast('platinum')
+    );
```

Comment:
nit: this could be simpler probably, but then also maybe we don't need these helper methods for each license type?

```suggestion
    return this.hasAtLeast('platinum');
```